packit/api.py

```diff
@@ -2139,6 +2143,107 @@ def run_osh_build(
 
         return cmd_result.stdout
 
+    def create_obs_project(
+        self,
+        project: str,
+        targets: str,
+        owner: Optional[str],
+        package_config: PackageConfig,
+        description: Optional[str],
+    ):
+        conf.get_config()
+        owner = (
+            owner or conf.config["api_host_options"][self.obs_helper._API_URL]["user"]
+        )
+        prj_name = project or f"home:{owner}:packit"
```

Comment:
A nit pick, but I would slightly prefer full names for variables.

```suggestion
        project_name = project or f"home:{owner}:packit"
```